src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    [`chip version`]: version,
```

Comment:
тут можна просто залишити version
```suggestion
    version,
```